src/main/java/subway/line/Line.java

```diff
@@ -49,8 +60,48 @@ public int getDistance() {
         return distance;
     }
 
+    public List<Station> getStations() {
+        return stations;
+    }
+
+    public List<Section> getSections() {
+        return sections;
+    }
+
     public void changeLineInfo(String name, String color) {
         this.name = name;
         this.color = color;
     }
+
+    public void changeDownStationId(SectionRequest sectionRequest) {
+        this.downStationId = sectionRequest.getDownStationId();
+        this.distance = sectionRequest.getDistance();
+    }
+
+    public boolean isExistStation(Station newStation) {
+        for(Station station : stations) {
+            if(newStation.getId() == station.getId()){
+                return true;
+            }
+        }
+
+        return false;
+    }
+
+    public void deleteStation(Long deleteStationId) {
+        if(sections.size() == 1) {
+            throw new IllegalArgumentException("구간이 1개 남은 경우 삭제할 수 없습니다.");
+        }
+
+        if(deleteStationId != downStationId){
```

Comment:
```suggestion
        if(!Objects.equals(deleteStationId, downStationId)){
```
요렇게 작성하면 조금 더 null에 안전한 코드가 됩니다 :)